cluster/cluster.go

```diff
@@ -267,6 +267,16 @@ type Cluster interface {
 	ClusterAlerts
 }
 
+//Secrets generic secret store operations
```

Comment:
Should I create separate package structure for each of PX API (secrets, schedule policy, service cmds etc) section moving into OSD or should they be clubbed together.